### PR TITLE
Store: Email remove MailChimp related config logic.

### PR DESCRIPTION
**Pull request by budzanowski**
### Information

Remove MailChimp related config checks. MailChimp is enabled on all configs so this checks make no sense anymore. I am not retiring the flag because it will be used for more store email development work.

### CODE REVIEW

File: config/production.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
 		"woocommerce/extension-promotions": true,
 		"woocommerce/extension-reviews": true,
 		"woocommerce/extension-settings": true,
-		"woocommerce/extension-settings-email": true,
+		"woocommerce/extension-settings-email": false,
```
Comment (timmyc) [inline config/production.json]: Doesn't this still need to be set to true to pass [this check](https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/index.js#L216)

File: config/stage.json [hunk 2]
```diff
@@ -139,7 +139,7 @@
 		"woocommerce/extension-promotions": true,
 		"woocommerce/extension-reviews": true,
 		"woocommerce/extension-settings": true,
-		"woocommerce/extension-settings-email": true,
+		"woocommerce/extension-settings-email": false,
```
Comment (timmyc) [inline config/stage.json]: Same as production... and in wpcalypso.

Comment (budzanowski) [reply config/stage.json]: Great catch. I should have seen that :/ It looks like it needs to be true and that I will need a separate flag for the general email settings work! Thanks.

Comment (justinshreve) [reply config/stage.json]: @budzanowski Does that mean this PR should be closed then?

Comment (budzanowski) [reply config/stage.json]: No. I just need to make a quick change and this will be good to go.

**Comment by budzanowski** [comment]
@timmyc Is this now good to go?

**Pull request merged**